auto_install/install.sh

```diff
@@ -1473,6 +1473,12 @@ askClientDNS(){
 
 			echo "pivpnDNS1=${pivpnDNS1}" >> ${tempsetupVarsFile}
 			echo "pivpnDNS2=${pivpnDNS2}" >> ${tempsetupVarsFile}
+
+			# Allow DNS incomming requests through UFW.
```

Comment:
I messed up the spelling of the comment, I spelled incoming with two m's. I thought I fixed it but it is still written with two m's.

Reply:
hmm ... Nope its actually correct ... https://github.com/pivpn/pivpn/blob/test/auto_install/install.sh#L1477

That is probably older version

Reply:
Strange, but fine :)

Reply:
@4s3ti @EWouters I’ve changed it, that this conversation is marked as resolved.